cpp_client/client.cpp

```diff
@@ -260,6 +263,19 @@ class FastRFSClient
     std::vector<sio::message::ptr> subscription_requests;
 };
 
+void make_new_connection(const std::vector<std::string> &token_pairs, const std::vector<int> &levels)  {
```

Comment:
can we make levels to be of double everywhere?

Reply:
OPTIONAL::

Reply:
resolving this, as it is not the scope of this change.